netlify-functions/src/main/scala/feral/netlify/HttpFunctionEvent.scala

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package feral.netlify
+
+import cats.data.NonEmptyList
+import io.circe.Decoder
+import io.circe.Encoder
+
+final case class HttpFunctionEvent(
```

Comment:
Let's add a test using this example:
https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-lambda.html

Reply:
Oh also let's add a `KernelSource` for this so the tracing middleware works too.